src/openrct2/Game.cpp

```diff
@@ -532,7 +532,12 @@ void save_game()
 {
     if (!gFirstTimeSaving)
     {
-        save_game_with_name(gScenarioSavePath.c_str());
+        char savePath[MAX_PATH];
```

Comment:
I am unsure if this is possible to be reached with an sv6 path but applied the same code anyway.